src/app/tree-grid/tree-grid-remote-paging-sample/tree-grid-remote-paging-sample.component.html

```diff
@@ -0,0 +1,33 @@
+<igx-tree-grid #treeGrid [data]="data | async" childDataKey="Content" expansionDepth="0" width="100%" height="550px"
```

Comment:
We can add an additional margin, so the grid does not get cut off on the top and on the bottom:
![image](https://user-images.githubusercontent.com/12784974/59175420-25ddf280-8b5e-11e9-89ad-abe8e6d859e2.png)

Reply:
Done.